core/core/test/serverspec/spec/kafka-exporter/kafka_exporter_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe 'Checking if kafka_exporter process is up' do
+  describe process('kafka_exporter') do
+    it { should be_enabled }
+    it { should be_running }
+  end
+end
+
+describe 'Checking user configuration' do
+  describe group('kafka_exporter') do
+    it { should exist }
+  end
+  describe user('kafka_exporter') do
+    it { should exist }
+    it { should belong_to_group 'kafka_exporter' }
+    it { should have_home_directory '/home/kafka_exporter' }
+    it { should have_login_shell '/sbin/nologin' }
```

Comment:
Should be: /usr/sbin/nologin
It'll work on RHEL7 because its /sbin is linked to /usr/sbin. On Ubuntu it's not linked (at least Azure images of 18.04).

Either way /usr/sbin should be safe for distros made in the last 5 years or so.

Reply:
Please note this change: https://github.com/epiphany-platform/epiphany/pull/97